ideascube/mediacenter/forms.py

```diff
@@ -19,12 +19,18 @@ class Meta:
         }
         exclude = ['package_id']
 
-    def clean_kind(self):
-        kind = self.cleaned_data['kind']
-        original = self.cleaned_data['original']
+    def clean(self):
+        cleaned_data = super().clean()
+
+        kind = cleaned_data['kind']
+        original = cleaned_data.get('original')
+
+        if original is None:
+            # The 'original' field validator already found this was wrong
```

Comment:
nit about the comment: original may also be None just because it was not given (in case of an update). ;)
Given the following lines of code, I'm not sure this check needs a comment actually, but in case we add one I'd suggest to fix a bit the wording accordingly. :)